techlibs/ecp5/Makefile.inc

```diff
@@ -11,6 +11,9 @@ $(eval $(call add_share_file,share/ecp5,techlibs/ecp5/bram.txt))
 $(eval $(call add_share_file,share/ecp5,techlibs/ecp5/arith_map.v))
 $(eval $(call add_share_file,share/ecp5,techlibs/ecp5/latches_map.v))
 
+$(eval $(call add_share_file,share/ecp5,techlibs/ecp5/abc_5g.box))
```

Comment:
Would you be so kind as to add the non 5g library too? And a `-device` option to `synth_ecp5`?
Selfishly, only because I don't have the 5g...

Reply:
I've been avoiding this because it means lots of staring at tables adding things together. Until we have clock constraints, I expect there's limited value as the grades are pretty much proportionally scaled anyway (for the purposes here).

Reply:
Sorry, I assumed it was something easy to do and you were the best man to do it!